docs/conf.py

```diff
@@ -10,21 +10,19 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import codecs
 import datetime
 import os
 import sys
 
 import edx_theme
 from unittest import mock
+from ..xblock import __version__
```

Comment:
@mraarif 
When I use relative import here the build fails raising an error that __name__ is. not is globals which I googled and found that we can't use relative imports in test files so I replaced it with "from XBlock.xblock" but then it raises error of "No module XBlock"
Any suggestions on it.....